Content.Server/_EE/Medical/CPR/CPRTrainingComponent.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+
+namespace Content.Server._EE.Medical.CPR;
+
+[RegisterComponent]
+public sealed partial class CPRTrainingComponent : Component
+{
+    [DataField]
+    public SoundSpecifier CPRSound = new SoundPathSpecifier("/Audio/Effects/CPR.ogg");
+
+    [DataField]
+    public TimeSpan DoAfterDuration = TimeSpan.FromSeconds(3);
+
+    [DataField] public DamageSpecifier CPRHealing = new()
+    {
+        DamageDict =
+        {
+            ["Asphyxiation"] = -6
+        }
+    };
+
+    [DataField] public float CrackRibsModifier = 1f;
+
+    [DataField] public float ResuscitationChance = 0.1f;
```

Comment:
This also does not seem to be respecting the relevant CVar, and we probably want it at zero.